xmlWorkbook.go

```diff
@@ -44,6 +44,70 @@ type xlsxWorkbook struct {
 	FileRecoveryPr      *xlsxFileRecoveryPr      `xml:"fileRecoveryPr"`
 }
 
+// MarshalXML implements xml.Marshaler
+func (x xlsxWorkbook) MarshalXML(e *xml.Encoder, start xml.StartElement) error {
+	x2 := struct {
```

Comment:
Would it be too costly to add these fields used in the custom marshal functions to the normal structs we already have and then populate the extra fields during a write? It seems like this exposes us to potential bugs where we might add other fields to things like `xlsxWorkbook` and forget to add them to these structs here and be very confused as to why they aren't appearing in the final output.